### PR TITLE
Update patch_release_template.md

### PR DESCRIPTION
**Pull request by gabi18**
Apply changes from 2.8.2 release, add link to template, remove tasks concerning beta program

### CODE REVIEW

File: .github/patch_release_template.md [hunk 1]
```diff
@@ -8,23 +16,25 @@
...
 ### Build
 
 * [ ] QA: Create [builds](https://confluence.owncloud.com/display/OG/Build+and+Tags#BuildandTags-Sprintbuild) for theme 'ownCloud' and 'testpilotcloud'
 * [ ] QA: Check the new subfolder of [testing download](https://download.owncloud.com/desktop/ownCloud/testing/) if `*tar.xz.asc` files are there. If not follow the [instructions](https://github.com/owncloud/enterprise/wiki/Desktop-Signing-Knowledge)
-* [ ] QA: Add a link to RC1 to https://owncloud.com/beta-testing/#desktop-beta @Marketing
-* [ ] QA: Inform [stakeholders](https://confluence.owncloud.com/display/OG/Marketing+and+Communication#MarketingandCommunication-InformStakeholdersAfterSuccessfulSprintBuild)
+* [ ] Beta/RC Communication (https://confluence.owncloud.com/display/OG/Marketing+and+Communication)
+   * [ ] Website links for beta (needed for the following posts)
+   * [ ] Central post https://central.owncloud.org/tags/c/news/desktop
+   * [ ] All other stakeholders
```
Comment (michaelstingl) [inline .github/patch_release_template.md:29]: specify?

File: .github/patch_release_template.md [hunk 2]
```diff
@@ -8,23 +16,25 @@
...
+
 
 ### QA
 
 #### DEV QA
 * [ ] DEV: [Smoke test](https://confluence.owncloud.com/display/OG/Manual+Tests#ManualTests-DEVSmokeTest)
 * [ ] DEV: Run [automated tests](https://confluence.owncloud.com/display/OG/Automated+Tests)
 #### BTR QA
-* [ ] QA: Inform subscribers of the beta program about a new version for testing
```
Comment (michaelstingl) [inline .github/patch_release_template.md:27]: Why not inform beta program subscribers?

Comment (gabi18) [reply .github/patch_release_template.md:27]: > Why not inform beta program subscribers?
 This is added to the "Beta/RC Communication" section

File: .github/patch_release_template.md [hunk 3]
```diff
@@ -8,23 +16,25 @@
...
 
 ### Build
 
 * [ ] QA: Create [builds](https://confluence.owncloud.com/display/OG/Build+and+Tags#BuildandTags-Sprintbuild) for theme 'ownCloud' and 'testpilotcloud'
 * [ ] QA: Check the new subfolder of [testing download](https://download.owncloud.com/desktop/ownCloud/testing/) if `*tar.xz.asc` files are there. If not follow the [instructions](https://github.com/owncloud/enterprise/wiki/Desktop-Signing-Knowledge)
-* [ ] QA: Add a link to RC1 to https://owncloud.com/beta-testing/#desktop-beta @Marketing
-* [ ] QA: Inform [stakeholders](https://confluence.owncloud.com/display/OG/Marketing+and+Communication#MarketingandCommunication-InformStakeholdersAfterSuccessfulSprintBuild)
+* [ ] Beta/RC Communication (https://confluence.owncloud.com/display/OG/Marketing+and+Communication)
+   * [ ] Website links for beta (needed for the following posts)
+   * [ ] Central post https://central.owncloud.org/tags/c/news/desktop
```
Comment (michaelstingl) [inline .github/patch_release_template.md:28]:
```suggestion
   * [ ] Central post https://central.owncloud.org/tags/c/news/desktop
   * [ ] Inform subscribers of the beta program about a new version for testing
```

File: .github/patch_release_template.md (hunk 1 above)
Comment (gabi18) [reply .github/patch_release_template.md:29]: The stakeholders are specified in above link https://confluence.owncloud.com/display/OG/Marketing+and+Communication

**Review by michaelstingl — APPROVED** [review]
LGTM

**Pull request merged**